test/DynamoCoreWpfTests/Properties/AssemblyInfo.cs

```diff
@@ -12,4 +12,5 @@
 [assembly: Guid("0e3a9cb1-69eb-4fc1-ab44-5fa36cfaa906")]
 [assembly: RequiresThread(ApartmentState.STA)]
 [assembly: InternalsVisibleTo("WpfVisualizationTests")]
-[assembly: InternalsVisibleTo("CrashReportingTests")]
\ No newline at end of file
+[assembly: InternalsVisibleTo("CrashReportingTests")]
+[assembly: TypeForwardedToAttribute(typeof(DynamoCoreWpfTests.Utility.DispatcherUtil))]
```

Comment:
why are there 2 typeforward attributes in 2 separate projects ?
Was this class defined twice in the 2 projects ?

Reply:
This class was used in both dll, because the original class is moved, both dlls affected needs to define type forward to the new dll the class moved to.

Reply:
I thought the typeforward attribute was only needed in the original assembly (where the type was defined), not in all assemblies that possibly referenced it . Let me check the docs

Reply:
At least that's how I interpret the docs: 
https://learn.microsoft.com/en-us/dotnet/standard/assembly/type-forwarding

Reply:
I double checked, yes you are right. Removed in the latest commit